content/guides/examples/_index.md

```diff
@@ -7,4 +7,4 @@ menu:
         weight: 2
 ---
 
-These are a few examples that cover various ways to use IPFS.
+These are a few examples that cover various ways to use libp2p.
```

Comment:
In sections like this, I would add something like a "Coming soon..." or "TODO".  The way it renders now, it looks like the section is just blank, which could be a browser problem.  It would be clearer to readers to just state "hey, this part is unfinished, but we're working on it."

![image](https://user-images.githubusercontent.com/225111/53033038-fecfa980-3424-11e9-850b-84d3bfa6f30c.png)

(I have the same suggestion on a few other unfinished pages, so I'll just flag those also.)